trino-lb-persistence/src/redis/mod.rs

```diff
@@ -25,6 +25,8 @@ use crate::Persistence;
 
 const LAST_QUERY_COUNT_FETCHER_UPDATE_KEY: &str = "lastQueryCountFetcherUpdate";
 
+const BINCODE_CONFIG: bincode::config::Configuration = bincode::config::standard();
```

Comment:
If they ever change the `standard()` to use fixed int encoding (for example), there will be some temporary disruption.
Probably not important.

Reply:
Thanks for pointing this out, I was also thinking about it...
To be honest I don't know what half of the options imply and figured they will likely have chosen defaults that perform well for the most cases (in terms of performance)